JS/index1.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        <title>ComIt </title>
+       
+    </head>
+
+<body>
+
+
+  <div id="divDate">  </div>
+  <script>
+    var div =  document.getElementById("divDate");
+    div.innerHTML = new Date().toDateString;
+  </script>
+  <script>
+      alert ('Hello, word!');
+  </script>
```

Comment:
You don't need to create multiple <script></script> tags.
You can write your whole code in one code script.

Reply:
Thanks!